README.md

```diff
@@ -86,16 +86,22 @@ when DataFusion might be be suitable and unsuitable for your needs:
   is primarily used directly by users as a serverless database and query system rather
   than as a library for building such database systems.
 
-- [pola.rs](http://pola.rs): Polars is one of the fastest DataFrame libraries at the time
-  of writing. Like DataFusion, it is also written in Rust but unlike DataFusion
-  it does not provide SQL nor many extension points.
+- [pola.rs](http://pola.rs): Polars is one of the fastest DataFrame
```

Comment:
I think its project name is `Polars`?

```suggestion
- [Polars](http://pola.rs): Polars is one of the fastest DataFrame
```

Reply:
![Screenshot 2023-01-16 at 8 06 10 PM](https://user-images.githubusercontent.com/490673/212787858-34af19e6-4fe5-4e33-bcb3-c2b5df236270.png)

That does appear to be the case 👍